Library/Homebrew/formula_support.rb

```diff
@@ -28,13 +28,14 @@ def valid?
 
   def to_s
     return @explanation unless @explanation.empty?
+    os_name = OS::Mac.os_name(ENV["HOMEBREW_OSX_VERSION"])
```

Comment:
I think I'd better revert this change to use `MacOS.os_name` unconditionally, and require `os/mac` on `test_formula_support`. Apparently the error message is meaningless on a Linux, isn't it?